src/plugins/data/common/search/aggs/buckets/lib/time_buckets/calc_es_interval.ts

```diff
@@ -12,7 +12,7 @@ import dateMath, { Unit } from '@elastic/datemath';
 import { parseEsInterval } from '../../../utils';
 
 const unitsDesc = dateMath.unitsDesc;
-const largeMax = unitsDesc.indexOf('M');
+const largeMax = unitsDesc.indexOf('w');
```

Comment:
Short term solution: Just keep this part of the fix (unless you want to remove this utility altogether)